### PR TITLE
[MIG] [16.0] product sticker

### PR DESCRIPTION
**Pull request by Shide**
Standard migration, added @rafaelbn as a maintainer too.

MT-6566 @moduon @rafaelbn @yajo @fcvalgar @Gelojr please review if you want :)

### CODE REVIEW

**Comment by fcvalgar** [comment]
> Standard migration, nothing changed.
> 
> [Moduon - Task #6609](https://www.moduon.team/web#id=6609&menu_id=256&cids=1&action=387&model=project.task&view_type=form) @moduon @rafaelbn @yajo @fcvalgar @Gelojr please review if you want :)

Internal task MT-6566

**Review by fcvalgar — APPROVED** [review]
Functional review,

LGTM thank you @Shide 

image

**Review by Gelojr — APPROVED** [review]
LGTM
image

**Review by yajo — APPROVED** [review]

**Review by rafaelbn — APPROVED** [review]
/ocabot merge patch

**Comment by rafaelbn** [comment]
- [x] @Shide we should be maintainers of this module please! 🙏🏼 😄 thank you! ❤️ 
- [x] After merge please review https://github.com/OCA/stock-logistics-reporting/pull/330

**Comment by Shide** [comment]
@etobella can you review and merge this one? You merge it on v15. Thanks! ❤️

File: product_sticker/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        "product",
+    ],
+    "data": [
+        "security/ir.model.access.csv",
+        "views/product_sticker_views.xml",
+        "views/product_attribute_views.xml",
+        "views/product_template_views.xml",
+        "data/menus.xml",
+    ],
+    "maintainers": ["Shide", "rafaelbn"],
```
Comment (etobella) [inline product_sticker/__manifest__.py:21]: why did you add @rafaelbn on the migration? You were already a maintainer :thinking:

Comment (Shide) [reply product_sticker/__manifest__.py:21]: I develop this module with @rafaelbn , under the umbrella of @moduon, so If I leave Moduon, someone needs to maintain this module and he is the right person to request improvements or approve changes.

I can do it in a separate PR when it's merged because I will be the maintainer in v16, but doesn't make sense to me if I'm the maintainer of the module right now and I consider acceptable to add @rafaelbn as a maintainer too.

**Comment by Shide** [comment]
@rousseldenis please can you merge this one? I'm the maintainer on v15. Thanks! ❤️

**Comment by rafaelbn** [comment]
Hello @OCA/product-maintainers ,

Please could you merge this PR? This is a direct migration from v15 of a module that we are maintainers 😄 :https://github.com/OCA/product-attribute/blob/15.0/product_sticker/__manifest__.py#L21


> Standard migration, added @rafaelbn as a maintainer too.

We have also pending [[MIG][16.0] stock_picking_report_product_sticker](https://github.com/OCA/stock-logistics-reporting/pull/330) wich depends on this one!

Thank you! ❤️ 🙏🏼

**Comment by yajo** [comment]
@Shide you're the maintainer in v15: https://github.com/OCA/product-attribute/blob/16284aff671fcba7e66191b44716c4afccdea60f/product_sticker/__manifest__.py#L21

Please try to issue the merge command yourself. [The bot should obey you if you're the previous version maintainer](https://github.com/OCA/oca-github-bot/issues/226#issuecomment-1603750889).

**Comment by rafaelbn** [comment]
Please @gurneyalex @legalsylvain could you please make a quick review here and merge? 🙏🏼 This is a direct migration of a module that we are maintainers 😄 ❤️

**Comment by HaraldPanten** [comment]
@Shide Aren't you mixing the pre-commit changes with the migration commit?

**Comment by Shide** [comment]
@HaraldPanten Yes, I'm mixing it because pre-commit doesn't give us relevant information

**Comment by etobella** [comment]
@Shide pre-commit may not give relevant information, but simplifies review, it exists for reviewers :wink:. Next time try to follow OCA rules please

**Comment by HaraldPanten** [comment]
> @HaraldPanten Yes, I'm mixing it because pre-commit doesn't give us relevant information

Hi Shide. I know, but pre-commit message has to be separate from migration. If you do it, I'll merge your PR ASAP 👍

**Comment by yajo** [comment]
Hi folks. When splitting the pre-commit commit, there's something you have to take into account. When the contributor changes the version or maintainers in the manifest, just doing `git commit` will rewrite the README. That's the thing that's happening here. This is because the README generator takes the manifest into account when deciding if it should update the README.

A very different thing is when, _before_ doing that, the contributor just runs `pre-commit run -a` and saves those automated reformats in a separate commit. However, that's not something applicable to this PR because if you do that _before_ changing anything, there won't be any changes anyways.

In short, if you follow [the recommended steps](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-16.0#technical-method-to-migrate-a-module-from-150-to-160-branch), you'll end up with a PR exactly like this one. Therefore, it's ready to merge.

**Comment by pedrobaeza** [comment]
That's not true, @Yajo. If you follow the guide you have linked (which is the official OCA guide for this), you end up with 2 commits, not one. The flag `--no-verify` on the commit command for the pre-commit auto-fixes makes that no additional changes are required to be done in the first commit due to linters.

If you want to change the current OCA guidelines, please raise an issue or mail on the usual channels to ask for that change. Meanwhile, you must follow the procedure and put the 2 commits, so @HaraldPanten's claim is totally legit.

**Comment by HaraldPanten** [comment]
> Hi folks. When splitting the pre-commit commit, there's something you have to take into account. When the contributor changes the version or maintainers in the manifest, just doing `git commit` will rewrite the README. That's the thing that's happening here. This is because the README generator takes the manifest into account when deciding if it should update the README.
> 
> A very different thing is when, _before_ doing that, the contributor just runs `pre-commit run -a` and saves those automated reformats in a separate commit. However, that's not something applicable to this PR because if you do that _before_ changing anything, there won't be any changes anyways.
> 
> In short, if you follow [the recommended steps](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-16.0#technical-method-to-migrate-a-module-from-150-to-160-branch), you'll end up with a PR exactly like this one. Therefore, it's ready to merge.

Hi Jairo, @rafaelbn @Shide  My only point here is trying to help you to move forward with this PR. All migrations are separating pre-commit stuff vs migration commit. That's why I asked you to do that.

You can see that we are doing this as well in every migration we do:

https://github.com/OCA/timesheet/pull/701
https://github.com/OCA/project/pull/1319
https://github.com/OCA/web/pull/2897
...

Once done, I'll merge this module. 🤙

**Comment by Shide** [comment]
@HaraldPanten I promise next migration PRs won't merge the pre-commit message.

But, I'm the maintainer and no one has modified this module in v15, so the commit author has been respected. 

I don't see any point to make me work more undoing a migration and make it again to have the same result.

I promise again in the next migration PRs won't merge the pre-commit message, but for now, I think it's enough work for that PR series.

Thanks in advance!

**Comment by HaraldPanten** [comment]
> @HaraldPanten I promise next migration PRs won't merge the pre-commit message.
> 
> But, I'm the maintainer and no one has modified this module in v15, so the commit author has been respected.
> 
> I don't see any point to make me work more undoing a migration and make it again to have the same result.
> 
> I promise again in the next migration PRs won't merge the pre-commit message, but for now, I think it's enough work for that PR series.
> 
> Thanks in advance!

Hi @Shide I'm not sure if I understood you well.

As far as I know, this is a standard migration. The only change is that you're adding Rafa as a maintainer. That's OK.

But... Why is it a problem to run pre-commit and have separate changes (pre-commit vs migration) in different commits?

It's not about me (or my rules), it's because everybody is doing the same in all OCA. And I'm asking these changes in every Migration PR I review. Please, have a look at --> https://github.com/OCA/account-financial-tools/pull/1819#issuecomment-2135202805

**Comment by Shide** [comment]
Closed in favor of https://github.com/OCA/product-attribute/pull/1698
